content/programming-cookbook/vs-code.md

```diff
@@ -0,0 +1,9 @@
+---
+title: VS Code
+showMetadata: true
+editable: true
+showToc: true
+order: 9
+---
+
+# VS Code
```

Comment:
I'm not sure this file should under `/programming-cookbook/vs-code` folder or not?

Reply:
@surrealist I add this for collapse menu like visual studio. For anyone have an idea of vscode.

![Screenshot 2021-04-26 161251](https://user-images.githubusercontent.com/37857209/116058854-7440ff80-a6aa-11eb-8210-fd84109cc0ba.png)

Reply:
So, I think if we have nothing about VS Code yet, 
we can remove it for now. Do you think so?

And from the image above, the menu structure seems odd to me.
I have some ideas on this so we can have a starting point for discussion.

### Choice 1
- Visual Studio
   - Tips & Tricks
      -  JSON to C# Classes
      -  _Another tip_

### Choice 2
- Visual Studio Tips & Tricks
   - JSON to C# Classes
   - _Another tips_

or choice 3 if we would like to gather all tricks & tips in a single file.

### Choice 3
- Visual Studio
  - Tips & Tricks
  - _Another topic_

How do you think?

Reply:
If you choose choice 1, I think it better moves "JSON to C# classes" into the "Tips & Tricks" since it is also a tip, right?

Anyway, what do you say, @aaronamm?

Reply:
@zhamppx97 Thank you so much for your contribution. I appreciate it.
@surrealist Thanks for you suggestion and supporting.

I need to checkout this PR to my local machine to play around about these options and I will be back to you soon. 
FYI, our system right now have some limit of nested items, I will put it in README as well.

Thanks.